kafka-connect-transforms/README.md

```diff
@@ -37,3 +37,17 @@ The SMT currently has no configuration. It can be used with the sink's CDC featu
 ```
 "iceberg.tables.cdcField": "_cdc_op",
 ```
+
+# DebeziumTransform
+_(Experimental)_
+
+The `DebeziumTransform` SMT transforms a Debezium formatted message for use by the sink's CDC feature.
+It will promote the `before` or `after` element fields to top level and add the following metadata fields:
+`_cdc_op`, `_cdc_ts`, and `_cdc_table`.
+
+## Configuration
+
+The SMT currently has no configuration. It can be used with the sink's CDC feature, e.g.
+```
+"iceberg.tables.cdcField": "_cdc_op",
```

Comment:
No it is not configurable in either, Debezium has a message spec that defines their format, and here we don't have a spec but just a convention.

Reply:
sure, as far as "Debezium has a message spec that defines their format" -- we are already transforming the dbz payload into a table structure (where in the payload there is an "op" field and we are promoting that into "_cdc_op" in the table (this would just be configuring what the table column to promote that "op" field into would be)

Reply:
This config is mostly for other cases when not using the transforms we're providing